lib/install/validate/validator.go

```diff
@@ -417,16 +417,18 @@ func (v *Validator) managedbyVC(ctx context.Context) {
 
 func (v *Validator) credentials(ctx context.Context, input *data.Data, conf *config.VirtualContainerHostConfigSpec) {
 	// empty string for password is horrific, but a legitimate scenario especially in isolated labs
-	if input.OpsPassword == nil {
+	if input.OpsCredentials.OpsPassword == nil {
 		v.NoteIssue(errors.New("Password for operations user has not been set"))
 		return
 	}
 
 	// check target with ops credentials
 	u := input.Target.URL
 
-	conf.Username = input.OpsUser
-	conf.Token = *input.OpsPassword
+	if input.OpsCredentials.OpsUser != nil {
+		conf.Username = *input.OpsCredentials.OpsUser
+	}
+	conf.Token = *input.OpsCredentials.OpsPassword
```

Comment:
hmm.. no, actually you tested the password, but do not for the user, which is slightly confusing.
I think you might just do the same thing for the opsuser name, how do you think?